README.md

```diff
@@ -29,7 +29,6 @@ Docker image publicly available on Docker Hub: https://hub.docker.com/r/cfplatfo
 |`repository.authenticate`   |out     |optional         | set to `in` to authenticate to a private repo for check and download only, `always` to authenticate to a private repository for upload, check and download.
 
 ### Deprecated parameters (since version 0.2.0)
-* `python_version`: gets mapped to `filename_match` if it's not a version number. `python_version` is now only used for the actual interpreter version to have a transparent mapping to pip.
```

Comment:
Not really deprecated according to the code: https://github.com/Pix4D/concourse-pypi-resource/blob/67c7d902ed0b0429f3bc68a86ef8870b2216aa39/pypi_resource/common.py#L82

Reply:
Note that if something is deprecated, it will be still accepted by the code, so the fact that the code supports it doesn't mean it is not deprecated...

Reply:
If you opened the link I posted, you clearly see which are the deprecated keys: https://github.com/Pix4D/concourse-pypi-resource/blob/67c7d902ed0b0429f3bc68a86ef8870b2216aa39/pypi_resource/common.py#L76

Reply:
I agree that I am maybe wrong here, but I in my understanding it is not deprecated and readme is a leftover

Reply:
Ah I see. The one line of the link was saying "available_keys". The "deprecated_keys" arrived later :-)

If the code one refers to is short, it helps the reviewer and avoids misunderstanding pasting also an extract directly in the comment.

Sometimes I copy the extract, put it into an editor, filter it through `cat -n` and paste it back in the comment (or in the chat). "cat -n" gives line numbers so it is easy to refer to them all in one place.